source/documentation/future.html.md

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
We don't need this page as it's already available on https://github.com/scalikejdbc/scalikejdbc.github.io/blob/develop/source/documentation/transaction.html.md

Reply:
Yes true! I removed this page